internal/network/network.go

```diff
@@ -0,0 +1,40 @@
+package network
+
+import "net"
+
+// Addresses returns the list of addresses for the local machine.
+// If ipv4 addresses are available, those are preferred over ipv6 ones.
+func Addresses() ([]string, error) {
```

Comment:
Without the Get feels more Go idiomatic, like net.Interfaces(), strings.Fields() etc.